lib/intl.ts

```diff
@@ -71,7 +72,7 @@ function getSlotLazy(obj: DateTimeFormatImpl, slot: LazySlot) {
 
 function createDateTimeFormat(
   dtf: DateTimeFormatImpl,
-  locale: Params['constructor'][0],
+  locale: Intl.LocalesArgument,
   optionsParam: Params['constructor'][1]
 ) {
   const hasOptions = typeof optionsParam !== 'undefined';
```

Comment:
Should the type of `options` one line below here be `typeof optionsParam` ?

Reply:
As far as I can tell it should not, because that could be undefined.